src/index.html

```diff
@@ -17,16 +17,38 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div
+      class="search-bar"
+      data-qa="big"
+    >
```

Comment:
Use the semantic tag here
```suggestion
    <form
      class="search-bar"
      data-qa="big"
    >
```